source/coap_security_handler.c

```diff
@@ -18,15 +18,24 @@
 #include "randLIB.h"
 #include "mbedtls/ssl_ciphersuites.h"
 
-
-const int PSK_SUITES[] = {
+#if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
+const int ECJPAKE_SUITES[] = {
     MBEDTLS_TLS_ECJPAKE_WITH_AES_128_CCM_8,
     0
 };
+#endif
+
+const static int PSK_SUITES[] = {
+    MBEDTLS_TLS_PSK_WITH_AES_128_CBC_SHA256,
+    MBEDTLS_TLS_PSK_WITH_AES_256_CCM_8,
+    MBEDTLS_TLS_PSK_WITH_AES_128_CCM_8,
+    0
```

Comment:
But those are living in same device, I renamed variable for ECJPAKE. This is a client/server list of supported ciphersuites in priority order (best in top). Maybe there are future use cases where we must support more ciphers and these lists can be updated. If we want an application to give this list, we can modify the interface later.